sheetScript.gs

```diff
@@ -4,6 +4,10 @@
 // MAYBE TODO: Filtering based on text (regexp?) in title and description
 // MAYBE TODO: NOT flags to include videos that are NOT from certain channels / do not contain text, etc
 
+var errorflag = false;
+var debugFlag_dontUpdateTimestamp = false;
+var debugFlag_dontUpdatePlaylists = false;
+var debugFlag_logWhenNoNewVideosFound = false;
```

Comment:
Should these be `constants`?

Reply:
Addressed in #48

Reply:
I left it as `var` since it would throw `TypeError` if they are `const`. Looking more into it, [this answer](https://stackoverflow.com/a/54413892) says to avoid consts due to it being not fully implemented correctly. We'll just stick with vars from here on.